test/integration/dialects/mysql/errors.test.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const chai = require('chai'),
```

Comment:
Use const per line

Reply:
We run eslint on tests and we have to change all those tests before we make this change. Its not very critical, its just a styling we have been following with new code base.